### ISSUE TITLE
"Выбор редакции"

### ISSUE THREAD

**Issue by Newbilius** [Newbilius/GamesRevival#27]
У средней игры обычно не более 3-5 портов-модов нужной тематики, но у монстров типа Doom/Quake их реально десятки, трудно понять, с чего лучше начать изучение.

Добавить блок "выбор редакции" на страницу с игрой, над списком портов? Или тег одноимённый? Надо подумать...

**Comment by shikulja** [comment]
Согласен, плюс профили на отдельных рецендентов, с сылкой на их обзор, или рекомендацию.
например на обзоры киномана авгна и пр.

**Comment by Newbilius** [comment]
@shikulja а ссылки на какие-то обзоры можно уже сейчас размещать - в конце описания мода предложить ссылки ;-)
